distribution/lib/Standard/Table/0.2.32-SNAPSHOT/src/Internal/Common_Helpers.enso

```diff
@@ -0,0 +1,21 @@
+from Standard.Base import all
+
+## PRIVATE
+
+   A helper used to select columns in both Standard.Table.Data.Table and Standard.Database.Data.Table
+
+   Arguments:
+   - columns: The current set of columns (must have a name property)
+   - column_list: The list of column names to filter down to
+   - reorder:
+select_columns : Vector -> Vector -> Boolean -> Vector
+select_columns = columns -> column_list -> (reorder=False) ->
+    mapper = x-> case x of
+        Text -> x
+        _ -> x.name
+    column_names = column_list.map mapper
+
+    new_columns = case reorder of
+        False -> columns.filter c->(column_names.contains c.name)
+        True -> column_names.map n->((columns.filter (c->c.name==n)).at 0)
+    new_columns
```

Comment:
I think you could rebase this branch on top of https://github.com/enso-org/enso/tree/wip/radeusgd%2Fcolumn-match-180855277 so that we could use the Matcher already.

Reply:
Also, I think we try to keep a convention that every file should end with a newline (as GH is showing here this red mark that there's no newline at end of file; most editors can be configured to add it automatically upon save).